src/requirements/VisitLink/VisitLinkRequirement.tsx

```diff
@@ -97,7 +100,7 @@ const VisitLinkRequirement = ({ ...props }: RequirementProps) => {
                 <Text wordBreak="break-word" flexGrow={1}>
                   <Original />
                 </Text>
-                {!!formContext && <ResetRequirementButton />}
+                <ResetRequirementButton />
```

Comment:
The condition was there so we only show the reset button in the edit drawer, so in normal view the admin sees the same as every other member. Is there a particular reason you've removed it, or was it just unclear to you? (In that case, we should add a comment about it)

Reply:
Other than this, approved

Reply:
Thanks, I believe it was accidental, whilst trying to figure out what it was doing. I added a comment to make it more clear!